### PR TITLE
[BugFix A11y] video tile a11y updates

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Updates the more options button in the video tile to announce
- muted state
- hand state
- speaking state
- call state (hold or calling)
- display name
# Why

Allows A11y users to better understand what a participant is doing when navigating through the VideoGallery

https://skype.visualstudio.com/SPOOL/_workitems/edit/3906226
https://skype.visualstudio.com/SPOOL/_workitems/edit/3906279
https://skype.visualstudio.com/SPOOL/_workitems/edit/3902007
https://skype.visualstudio.com/SPOOL/_workitems/edit/3906110
# How Tested

Validated locally 
image
image
image

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/utils.ts [hunk 1]
```diff
@@ -410,3 +411,37 @@ export const isEnterKeyEventFromCompositionSession = (e: KeyboardEvent): boolean
...
+  displayName?: string,
+  isMuted?: boolean,
+  isHandRaised?: boolean,
+  state?: string,
+  isSpeaking?: boolean,
+  strings?: VideoTileStrings
+): string => {
+  let ariaDescription = '';
+  if (displayName) {
+    ariaDescription = ariaDescription += ` ${displayName}`;
```
Comment (JamesBurnside) [inline packages/react-components/src/components/utils.ts:428]: String concatenation with localized strings is bad as languages don't piece together the same. Could we instead do something like:
```ts
/// strings.json
"moreOptionsButtonAriaLabel" : "More Options {participantName} {participantMutedStatus} {participantRaisedHandStatus} {participantIsSpeakingStatus}"
```

**Review by JamesBurnside — CHANGES_REQUESTED** [review]

File: packages/react-components/src/components/VideoTile.tsx [hunk 2]
```diff
@@ -213,23 +222,56 @@ const videoTileMoreMenuProps = {
...
     return <></>;
   }
 
   const optionsIcon = canShowContextMenuButton ? 'VideoTileMoreOptions' : undefined;
-
   return (
     <IconButton
       data-ui-id="video-tile-more-options-button"
-      ariaLabel={strings?.moreOptionsButtonAriaLabel}
+      ariaLabel={locale.moreOptionsButtonAriaLabel + ` ${moreButtonAiraDescription}`}
```
Comment (edwardlee-msft) [inline packages/react-components/src/components/VideoTile.tsx:274]:
```suggestion
      ariaLabel={locale.moreOptionsButtonAriaLabel + ` ${moreButtonAriaDescription}`}
```

**Review by edwardlee-msft — APPROVED** [review]

File: packages/react-components/src/components/VideoTile.tsx (hunk 2 above)
Comment (edwardlee-msft) [reply packages/react-components/src/components/VideoTile.tsx:274]: nit, strong concat can be dangerous with locale stuff. So make sure this makes coherent sense!

File: packages/react-components/src/components/utils.ts (hunk 1 above)
Comment (JamesBurnside) [reply packages/react-components/src/components/utils.ts:428]: Not sure if there's a way to mark the templated string as optional? cc @mgamis-msft

File: packages/react-components/src/components/utils.ts [hunk 3]
```diff
@@ -410,3 +412,44 @@ export const isEnterKeyEventFromCompositionSession = (e: KeyboardEvent): boolean
...
+export const formatMoreButtonAriaDescription = (
+  displayName?: string,
+  isMuted?: boolean,
+  isHandRaised?: boolean,
+  state?: string,
+  isSpeaking?: boolean,
+  strings?: VideoTileStrings
+): string => {
+  let mutedState;
+  let handRaisedState;
```
Comment (JamesBurnside) [inline packages/react-components/src/components/utils.ts:428]: nit: use const and assign up front:
```
const mutedAria = isMuted ? strings?.moreOptionsParticipantMutedStateMutedAriaLabel : strings?.moreOptionsParticipantMutedStateUnmutedAriaLabel;
const handRaisedAria = isHandRaised ? strings?.moreOptionsParticipantHandRaisedAriaLabel : undefined;
```

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**